app/public/js/isWrapped.js

```diff
@@ -0,0 +1,11 @@
+(function (global) {
+  'use strict';
+  var $ = global.jQuery;
+  $('h2.results__name').each(function(){
+    var $height = $(this).height();
+    var $lineHeight = Math.floor(parseInt($(this).css('line-height').replace('px','')));
```

Comment:
Same as above, the variable can just be `lineHeight`.
Can you add the radix to the parseInt call please e.g.
`var lineHeight = Math.floor(parseInt($(this).css('line-height').replace('px',''), 10));`
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt for more info.